pkg/backup/restore_planning.go

```diff
@@ -91,6 +91,14 @@ var featureRestoreEnabled = settings.RegisterBoolSetting(
 	featureflag.FeatureFlagEnabledDefault,
 	settings.WithPublic)
 
+var restoreCompactedBackups = settings.RegisterBoolSetting(
+	settings.ApplicationLevel,
+	"restore.compacted_backups.enabled",
+	"allow restoring from compacted backups",
+	false,
```

Comment:
i think we should leave this default true, so only the compaction threshold is the knob users need to turn.

Reply:
Ah yeah that's reasonable 👍